src/common/tools/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+  "parser": "@typescript-eslint/parser",
```

Comment:
What are these changes?

Reply:
I was getting warnings/issue locally about the TS parser for the linting wasn't specified. It appears atleast my IDE doesn't walk up the tree to find missing parameters if `.eslintrc.json` exists in a directory.